templates/core/terraform/.terraform.lock.hcl

```diff
@@ -58,21 +58,21 @@ provider "registry.terraform.io/hashicorp/local" {
 }
 
 provider "registry.terraform.io/hashicorp/random" {
-  version = "3.1.2"
```

Comment:
Does this need to change?

Reply:
Removed - must have been updated by mistake - thanks! :-)